components/base/Typography.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import classnames from "classnames";
+import { POPPINS, ROBOTO } from "../typography";
+import {
+  HEADLINE_TEXT,
+  PARAGRAPH_TEXT,
+  WHITE_PRIMARY,
+  GREY_5,
+  GREY_4,
+} from "../colors";
+
+// Typography function
+function Typography({ text, styleVariant, tag, isDark }: TypographyProps) {
+  return (
+    <TypographyStyles
+      as={`${tag}`}
+      className={classnames(`${styleVariant}`, { dark: isDark })}
+    >
+      {text}
+    </TypographyStyles>
+  );
+}
+
+// Typography prop types
+type TypographyProps = {
+  text: string;
+  styleVariant:
```

Comment:
I wonder if we should use [composition](https://emotion.sh/docs/composition) or something to split these style variants into their own components that share some common CSS.

e.g.

```typescript
const headingStyles = css`/* heading styles */`
export const H1 = ({ children }) => <h1 css=[headingStyles, css`/* h1 styles */`]>{children}</h1>
```

Reply:
I like that idea! I'll get those updated 👨‍⚖️